package.json

```diff
@@ -47,18 +47,17 @@
     "fr-FR": "./languages/fr.json"
   },
   "devDependencies": {
-    "@peertube/peertube-types": "^4.2.2",
-    "@tsconfig/node16": "^1.0.3",
-    "@typescript-eslint/eslint-plugin": "^5.33.0",
-    "@typescript-eslint/parser": "^5.33.0",
-    "esbuild": "^0.14.36",
-    "eslint": "^8.21.0",
-    "eslint-config-standard-with-typescript": "^22.0.0",
+    "@peertube/peertube-types": "^7.0.0",
```

Comment:
There was a good reason to let `"@peertube/peertube-types": "^4.2.2",`: in the `engine` section, I mention that this plugin is compatible with `"peertube": ">=4.2.0"`.
So we must use the Peertube 4.2.x types, to be sure to not use features that are not avaible.
This is explained in the README.